.eslintignore

```diff
@@ -1,3 +1 @@
-**/lib/**
```

Comment:
Do the `assets` and `lib` directory that 11ty copies to not need to be ignored?

Reply:
@jobara not anyore— the entire `dist` directory is in `.gitignore` and so the copied files will be ignored as a result. The old method put the Infusion sources into `src/lib` as a temporary destination and this was version controlled, which meant it had to be explicitly ignored.